exercises/binary-search/instructions.md

```diff
@@ -0,0 +1,28 @@
+# Instructions
+
+Your task is to implement a binary search algorithm.
+
+A binary search algorithm sorts a list by repeatedly splitting it in half, only keeping the half which contains the item we're looking for.
```

Comment:
```suggestion
A binary search algorithm finds an item in a list by repeatedly splitting it in half, only keeping the half which contains the item we're looking for.
```

I don't think I could truthfully say that binary search sorts the list; instead, the list already has to be sorted (that fact is already mentioned below)